Source: written by Colin Walters <walters@verbum.org>
Upstream: yes, commited in Gnome Git
Reason: Fix build of DBusGLib-1.0.gir

From 9728c5ad69e4922e192cc279789aa0db08b45dca Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 23 Oct 2009 02:57:13 +0000
Subject: [DBusGLib] Fix up hand-written gir for format changes

---
diff --git a/gir/DBusGLib-1.0.gir b/gir/DBusGLib-1.0.gir
index d79a018..41d273a 100644
--- a/gir/DBusGLib-1.0.gir
+++ b/gir/DBusGLib-1.0.gir
@@ -3,7 +3,8 @@
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0"
 	    xmlns:glib="http://www.gtk.org/introspection/glib/1.0">
-  <include name="GObject"/>
+  <include name="GObject" version="2.0"/>
+  <package name="dbus-glib-1"/>
   <namespace name="DBusGLib" version="1.0" shared-library="dbus-glib-1">
     <record name="Connection" c:type="DBusGConnection"/>
     <record name="MethodInvocation" c:type="DBusGMethodInvocation"/>
--
cgit v0.8.3.1

From 955452107bd87eecd5c5af3f6552c650fc49bdf2 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 23 Oct 2009 03:01:07 +0000
Subject: [DBusGLib] Add transfer-ownership

---
diff --git a/gir/DBusGLib-1.0.gir b/gir/DBusGLib-1.0.gir
index 41d273a..c50bfbb 100644
--- a/gir/DBusGLib-1.0.gir
+++ b/gir/DBusGLib-1.0.gir
@@ -12,16 +12,14 @@
       glib:type-name="DBusGProxy" glib:get-type="dbus_g_proxy_get_type">
     </class>
     <record name="ProxyClass" c:type="DBusGProxyClass"/>
-
-    <function name="connection_open" c:identifier="dbus_g_connection_open"
-     throws="1">
+    <function name="connection_open" c:identifier="dbus_g_connection_open" throws="1">
       <return-value transfer-ownership="full">
         <type name="Connection" c:type="DBusGConnection*"/>
       </return-value>
       <parameters>
-        <parameter name="address">
-	  <type name="utf8" c:type="char*"/>
-	</parameter>
+        <parameter name="address" transfer-ownership="none">
+	      <type name="utf8" c:type="char*"/>
+        </parameter>
       </parameters>
     </function>
   </namespace>
--
cgit v0.8.3.1

